src/components/ProjectAssignModal/index.tsx

```diff
@@ -80,7 +85,12 @@ function ProjectAssignModal<T extends ProjectAssignableEntity>(props: ProjectAss
       ]}
     >
       <Grid item xs={12} textAlign='left'>
-        <ProjectsDropdown availableProjects={availableProjects} record={entity} setRecord={handleUpdateProject} />
+        <ProjectsDropdown
+          availableProjects={availableProjects}
+          record={entity}
+          setRecord={handleUpdateProject}
+          allowUnselect
```

Comment:
allowUnselect only when onUnAssign is passed in?

Reply:
Ah, good catch!